src/share/rsdk/configs/socs.json

```
@@ -75,6 +75,9 @@
         "soc_family": "qualcomm",
         "partition_table_type": "gpt",
         "firmware_type": "edk2",
+        "soc_install_recommends" :[
+            "qcs6490"
+        ],
```

```suggestion
        "task_install_recommends" :[
            "jammy",
            "noble"
        ],
```
我在[飞书上](https://applink.feishu.cn/client/message/link/open?token=AmVUjoJLkYAEaMks1fTHwAM%3D)说的是这个意思，这样core里面的判断就可以简化了。

本质上是否可以安装recommend packages是跟distro版本相关的，跟soc是个间接的关系

然后可以给其他几个family补上对应的code name，参考products.json里面支持的版本